### DISCUSSION TITLE
Something went wrong Expecting value: line 1 column 1 (char 0)

### DISCUSSION THREAD

**Discussion by aimartyr** [oobabooga/text-generation-webui#352] [Q&A]
<img width="1247" alt="image" src="https://user-images.githubusercontent.com/125782068/225513367-cfb362c1-5e75-410a-854d-52cfdb66fdac.png">
 I am at a loss as to what to do about this problem.

**Comment by jiwenbo12** [comment]
I also encountered the same situation.But I have no way to observe through the command line where the problem is. I hope the author can provide some help.

**Reply by ClayShoaf** [reply]
Why are you not able to look at the commandline? Or are you just saying that no error messages are appearing there?

**Comment by jiwenbo12** [comment]
I tried to restart my computer and found that the program can run normally. You can try to restart your computer and reopen the web UI.
